src/App.js

```diff
@@ -14,11 +17,94 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: goodsFromServer,
+    wasStarted: false,
+    wordLength: 1,
+  }
+
+  start = () => this.setState({ wasStarted: true });
+
+  reset = () => this.setState({
+    goods: goodsFromServer,
+    wordLength: 1,
+  });
+
+  reverseList = () => (
+    this.setState(state => ({
+      goods: [...state.goods].reverse(),
+    }))
+  );
+
+  sortAlphabetically = () => (
+    this.setState(state => ({
+      goods: [...state.goods].sort(),
+    }))
```

Comment:
for alphabetically sorting is a little better to use localeCompare

Reply:
why didn't you fix this?

Reply:
cuz i agree that localeCompare is a best practice, but in this task, it was not necessary. i fixed it now